packages/orchestrator-ui-components/src/components/WFOSubscription/index.ts

```diff
@@ -1,9 +1,8 @@
 export * from './utils';
-export * from './WFOFixedInputBlock';
 export * from './WFOProcessesTimeline';
-export * from './WFOProductBlock';
-export * from './WFOSubscriptionActions';
-export * from './WFOSubscriptionBlock';
-export * from './WFOSubscriptionDetailTree';
-export * from './WFOSubscriptionGeneral';
+export * from './SubscriptionProductBlock';
```

Comment:
Again, I'm fine with renames; but I wonder about the rationale; and it might be good to add it to the ADR/Naming convention rules.

Reply:
same comment